### ISSUE TITLE
[Bug]: Object deletion works unstable.

### ISSUE THREAD

**Issue by ghost** [Yimura/YimMenu#1395]
### Describe the bug

I remember that Yimmenu had a better item deletion ability before.
In the current version, it cannot delete fixed objects (like lighting poles) and objects left by other modders.

### Steps To Reproduce

Context menu, and use delete feature in Weapon menu

### Expected Behavior

Deletion should delete fixed objects such as lampposts and objects created by other modders

### Screenshots and/or Logs

[log.txt](https://github.com/YimMenu/YimMenu/files/11713288/log.txt)

https://imgur.com/a/gu7tk1T

### Store

Epic Games

### Additional context

Context menu boxes seem a bit inconsistent for fixed objects.

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

**Comment by BUZZARDGTA** [comment]
Also we can't delete the cage spawned from the custom weapon option.

**Comment by Yimura** [comment]
I don't think we can delete fixed map objects, however we should be able to remove cages.

What I see from the video is that the context menu does recognise the fixed object (fire hydrant) but fails to remove it.
If you drive with against the object will make it detached from the world and allow you to delete it.

**Comment by ghost** [comment]
> I don't think we can delete fixed map objects, however we should be able to remove cages.
> 
> What I see from the video is that the context menu does recognise the fixed object (fire hydrant) but fails to remove it. If you drive with against the object will make it detached from the world and allow you to delete it.

Not every object left by other modders can be removed. For example, tents or vehicles in unrelated places.
I know that fixed objects are not deleted, but as far as I remember previously, objects that could be plucked from the ground, such as light poles, could be removed without intervention.
